module/uninstall.sh

```
@@ -0,0 +1,232 @@
+# Expire active thaw TTL in greezer and re-enable freezer monitoring
+if [ -n "$GMS_UID" ]; then
+    cmd greezer thuid "$GMS_UID" 0 2>/dev/null || true
+    cmd greezer monitor "$GMS_UID" 2>/dev/null || true
+fi
+
+# ==============================================================================
+# 6. Restore System & Secure Settings to Stock State
+# ==============================================================================
+if [ -f "$STOCK_CONF" ]; then
+    while IFS= read -r line || [ -n "$line" ]; do
+        line="${line%$'\r'}"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, $'..' is undefined.

(SC3003)